### DISCUSSION TITLE
Does this software offer account synchronization?

### DISCUSSION THREAD

**Discussion by beedell-roke** [learn-awesome/learndb#81] [Q&A] [answered]
https://socialhub.activitypub.rocks/t/learnawesome-is-an-activitypub-compliant-alternative-to-goodreads/764 is somewhat confusing me - although I see a clear indication that I should be able to register somewhere, I am unable to locate it. I've filed https://github.com/learn-awesome/learndb/issues/80#issue-2095130973 to remediate this should my presumption be inaccurate.

**Comment by nileshtrivedi** [comment, marked as answer]
@RokeJulianLockhart That refers to the previous version of LearnAwesome which was a Ruby on Rails application - with a backend and database with user accounts and ActivityPub support. I found it difficult to maintain as not enough contributors came forward (Ruby seems to have lost steam over the years).

You can find the source-code for that version here: https://github.com/learn-awesome/learn
Once we switched to a static site model, I requested people at https://fediverse.party/ to remove LearnAwesome from their catalog as it no longer supports ActivityPub.
